docs/design/DB/dokuha-roadmap.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE Users (
+	user_id SERIAL PRIMARY KEY,
+	username VARCHAR(255) NOT NULL,
+	password VARCHAR(255) NOT NULL
+);
+
+
+
+CREATE TABLE Contents (
```

Comment:
POstgreSQL で大文字使う意味あんまないから、小文字に統一しよう！
https://avinton.com/academy/database-naming-conventions/
> Postgresでは小文字も大文字もすべて小文字として処理されます。

```suggestion
CREATE TABLE contents (
```